pytorch_lightning/core/saving.py

```diff
@@ -17,7 +17,7 @@
 try:
     from omegaconf import OmegaConf
 except ImportError:
-    Container = None
+    pass
```

Comment:
I think you want `OmegaConf = None` here, otherwise you are accessing an uninitialized variable.

Reply:
ah you're right, forgot after removing `Container` to replace this